sycl/source/detail/kernel_bundle_impl.hpp

```diff
@@ -367,13 +367,15 @@ class kernel_bundle_impl {
         detail::getSyclObjImpl(*It);
 
     RT::PiKernel Kernel = nullptr;
-    std::tie(Kernel, std::ignore) =
+    const KernelArgMask *ArgMask = nullptr;
+    std::tie(Kernel, std::ignore, ArgMask) =
```

Comment:
```
auto [Kernel, IgnoreName, ArgMask] = ...
```

Reply:
On second thought, I prefer the std::tie version since it allows to ignore the value without creating an unused variable.